tests/test_util.cuh

```diff
@@ -30,6 +30,10 @@
 #include "merlin/utils.cuh"
 #include "merlin_hashtable.cuh"
 
+#define UNEQUAL_EXPR(expr1, expr2)                            \
```

Comment:
I perfer this( sorry for no formatted) :
`#define UNEQUAL_EXPR(expr1, expr2)                            \`
`  {std::cout <<  __FILE__ << ":" << __LINE__ << ":Unequal\n"   \`
`         << "\t\t" << #expr1 << " != " << #expr2 << "\n"; }`

Reply:
OK, I will update. (^▽^ )